packages/tools/client-debugger/client-debugger-view/src/DevtoolsView.tsx

```diff
@@ -438,10 +440,37 @@ function ContainersMenuSection(props: ContainersMenuSectionProps): React.ReactEl
 		);
 	}
 
-	// TODO: add button to refresh list of containers
 	return (
-		<MenuSection header="Containers" key="container-selection-menu-section">
-			{containerSectionInnerView}
-		</MenuSection>
+		<div>
+			<ContainerMenuSection header="Containers">{RefreshButton()}</ContainerMenuSection>
```

Comment:
Don't you need to pass in a 'children' prop? Or is that represented by `{RefreshButton()}`

Reply:
No. This refresh button will post a get container list request message. When the new container list message comes in the menu view will render again due to the useEffect() detect a new message.